### DISCUSSION TITLE
Shouldn't the `5.0.1` release be `5.1.0`?

### DISCUSSION THREAD

**Discussion by julealgon** [AutoMapper/AutoMapper.Extensions.OData#216] [Q&A]
Just saw [the release notes for 5.0.1](https://github.com/AutoMapper/AutoMapper.Extensions.OData/releases/tag/v5.0.1), which includes a feature addition:
> Add support for EnableConstantParameterization by @alexmo16 in https://github.com/AutoMapper/AutoMapper.Extensions.OData/pull/215

If the project is following semantic versioning, it should've been tagged `5.1.0` instead of `5.0.1`.

**Comment by BlaiseD** [comment]
I think you're right it's closer to a feature.  Exposing a flag also passes as a patch I think.
